charts/tempo-distributed/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 name: tempo-distributed
 description: Grafana Tempo in MicroService mode
 type: application
-version: 1.9.9
+version: 1.9.10
```

Features should be bumped to minor version.

```suggestion
version: 1.10.0
```